apps/wing/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "build": "tsc && pnpm copy-root-readme",
-    "eslint": "eslint --ext .ts src",
+    "eslint": "eslint --fix --ext .ts src",
```

Comment:
I snuck this change into your PR as part of settling the merge conflict with a recent PR adding eslint to this project